### ISSUE TITLE
Highlighters

### ISSUE THREAD

**Issue by moshpirit** [andstatus/andstatus#374]
Highlight some posts with filtered words. (e.g. Andstatus → all the notes with "Andstatus" will be highlighted)

See #161

**Comment by jankusanagi** [comment]
You miss that from Dianara, don't you? =)

**Comment by moshpirit** [comment]
Yes, haha :P

**Comment by yvolk** [comment]
Yes, we are really accustomed to contextual highlighting now. Highlighting search words looks natural and even as an expected behavior :-)
